src/adminBase.js

```diff
@@ -124,24 +108,27 @@ export default class AdminBase extends HasModels {
     let inst = {}
     if(this.populate && this.populate.length > 0) 
       for (let pop of this.populate) inst[pop] = {}
-    return this.templater.render(this.prefix+'-form', {
+    return this.templater.render(this.opts.template_prefix+'-form', {
       req,
-      base: req.adminOpts.basePath+this.base,
+      base: req.adminOpts.basePath+this.opts.base,
       user: req.user,
       title: 'New '+this.constructor.name,
-      inst
+      inst,
+      name: this.opts.display_name,
+      attributes: this._getAttrs(this.models.model)
     })
   }
 
   _delete (req, res) {
     return this.models.model.destroy(req.params.id).then((inst) => {
-      req.flash('info', this.display_name()+' deleted');
+      req.flash('info', this.opts.display_name+' deleted');
       res.redirect(req.adminOpts.basePath+this.base)
     })
   }
 
   save (req, res) {
-    return this._save(req, res)
+    if(this.opts.save) return this.opts.save(req, res, this)
```

Comment:
I see, you're handling that particular override case, but the downside to not extending is right there in `this` - at least bind this.opts.save rather than giving it a changed signature?